src/applications/personalization/profile/util/contact-information.js

```diff
@@ -0,0 +1,84 @@
+import { ADDRESS_POU, USA } from '@@vap-svc/constants';
+import ADDRESS_DATA from '~/platform/forms/address/data';
+import pickBy from 'lodash/pickBy';
+
+/**
+ * Returns a copy of the input object with an added `view:livesOnMilitaryBase`
+ * value if the address is a overseas military mailing address
+ *
+ */
+const livesOnMilitaryBase = data => {
+  if (
+    data?.addressPou === ADDRESS_POU.CORRESPONDENCE &&
```

Comment:
I cannot for the life of me remember why we bother checking to see if this is a mailing address as part of this check. And why we don't just check that the `data.addressType` is `ADDRESS_TYPES.OVERSEAS_MILITARY` 
<img width="1030" alt="Screen Shot 2020-12-15 at 9 05 58 AM" src="https://user-images.githubusercontent.com/20728956/102248073-9bdeba80-3eb5-11eb-88c3-5294bd2adcf7.png">

Reply:
Thanks for catching that! I have made a note of this an I will address this in one of my next PRs where I get more into the nitty gritty of the logic instead of just moving stuff around.